sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: 2.0.4413
```

Comment:
This file should be in /eng/mgmt/mgmtmetadata

Reply:
Updated, please review

Reply:
I think it's accidentally removed...

Reply:
That is weird, I moved it to /eng/mgmt/mgmtmetadata but it's not showing up in the diff here

Reply:
Updated

Reply:
Please review again